.github/workflows/packageOnPrem.yml

```diff
@@ -0,0 +1,50 @@
+name: Package vsix
+
+on:
+  workflow_dispatch:
+    inputs:
+      binaryVersion:
+        type: string
+        description: the version of the tabnine binary which will be packaged into this vsix file. it assumes it has been built and available to download
+        required: true
+      vsixVersion:
+        type: string
+        description: the version to tag this vsix with. should be valid semver format x.y.z
+        required: true
+
+jobs:
+  build:
+    name: Make vsix
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - uses: actions/setup-node@v3
+        with:
+          node-version: '16.19'
+      #          runs the binaries downloader script which will be packaged into this plugin
+      - name: pull binaries
+        run: VERSION=${{ inputs.binaryVersion }} ./binaries/downloadBinaries.sh
+      #      https://docs.github.com/en/actions/learn-github-actions/contexts#github-context
+      - name: modify readme
+        run: |
+          echo "" > ./README.md
```

Comment:
Why deleting the current Readme and not just adding to it?

Reply:
because it has lots of links and junk